rllib/policy/torch_policy.py

```diff
@@ -104,9 +107,12 @@ def __init__(
         """
         self.framework = "torch"
         super().__init__(observation_space, action_space, config)
-        if torch.cuda.is_available() and ray.get_gpu_ids():
+        if ray.get_gpu_ids():
```

Comment:
In TorchPolicy: Decide inside Policy constructor, whether to place the Model on the GPU.